EasyPost.Tests/AddressTest.cs

```diff
@@ -246,6 +246,18 @@ public void TestInstanceCreateWithVerifications()
         [ExpectedException(typeof(HttpException))]
         public void TestRetrieveInvalidId() => Address.Retrieve("not-an-id");
 
+        [TestMethod]
+        public void TestRetrieveAll()
+        {
+            AddressCollection addressCollection = Address.All();
+            Assert.IsNotNull(addressCollection);
+            if (addressCollection.addresses.Count > 0)
```

Comment:
This is fine, could also iterate the returned list and assert against each

Reply:
Is this blocking the merge?

Reply:
Nope, if I stamp it means we can proceed but any feedback regardless of who left it is highly encouraged to include 😉 I won't stop progress for little nit picks though. Some of these are learnings we can share as a team for the future.

Reply:
Resolved in 7e048a5